panther_gazebo/src/led_strip.cc

```diff
@@ -0,0 +1,97 @@
+// Copyright 2024 Husarion sp. z o.o.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <iostream>
+
+#include <yaml-cpp/yaml.h>
+#include <gz/math.hh>
+#include <gz/msgs.hh>
+#include <gz/transport.hh>
+
+#include <panther_gazebo/led_strip.hpp>
+
+LedStrip::LedStrip(
+  const gz::math::Vector3d & position, float led_strip_width, const std::string & topic,
+  int start_id)
+: position_(position), led_strip_width_(led_strip_width), topic_(topic), start_id_(start_id)
+{
+  gz::transport::SubscribeOptions opts;
+  opts.SetMsgsPerSec(10u);  // Setting to high frequency caused lags
+  node.Subscribe(topic_, &LedStrip::imageCallback, this, opts);
+}
+
+void LedStrip::imageCallback(const gz::msgs::Image & _msg)
+{
+  gz::msgs::Marker_V markerMsgs;
```

Comment:
```suggestion
  gz::msgs::Marker_V marker_msgs;
```
Use snake case for variables. Applies to other files.